specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/examples/AssessmentsMetadata/CreateAssessmentsMetadata_subscription_example.json

```diff
@@ -0,0 +1,39 @@
+{
+  "parameters": {
+    "api-version": "2019-01-01-preview",
+    "subscriptionId": "0980887d-03d6-408c-9566-532f3456804e",
+    "assessmentMetadataName": "ca039e75-a276-4175-aebc-bcd41e4b14b7",
+    "assessmentMetadata": {
+      "properties": {
```

Comment:
All of these fields are marked as readonly, so the sample shouldn't include them (or the model should be updated to allow writable fields).

Reply:
updated the model :)